### DISCUSSION TITLE
How do I get the Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by muhammadnurulahsan** [Schweinepriester/github-profile-achievements#484] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84895237/201622135-3420f9a8-4df7-4ffe-9e1e-d1b9bd196a1f.png)
How do I get the Galaxy Brain badge?

**Comment by d4tdev** [comment, marked as answer]
U need to answer 2 question, if it's true, maybe post owner will "mark as answer". U can take Galaxy Brain  by this

**Reply by d4tdev** [reply]
u can see the questions here
https://github.com/orgs/community/discussions
